src/T2DMap.cpp

```diff
@@ -2351,7 +2351,7 @@ void T2DMap::paintMapInfo(const QElapsedTimer& renderTimer, QPainter& painter, c
 
 void T2DMap::mouseDoubleClickEvent(QMouseEvent* event)
 {
-    if (mDialogLock) {
+    if (mDialogLock || (event->buttons() & Qt::RightButton)) {
         return;
```

Comment:
Given that we are rejecting the event I think we need to call:
```cpp
    event->ignore();
```
before we hit the `return;` to flag that we have NOT used the event (so it gets passed up to the parent widget).

Related - we also should be calling `event->accept();` before we leave the bottom of the method. As it happens the `accepted` flag will be set or reset automagically by the default Qt code that this event handler is overriding - but it is not clear to me which state the base class will be using (and so could be skipped here).